lib/inferno/dsl/fhir_evaluator/evaluator_util.rb

```diff
@@ -0,0 +1,81 @@
+module Util
+  # General util methods
+  def self.get_meta_profile(resource)
```

Comment:
These should live in a module with a real name which is included in the places it's needed.